docs/appendices/release-notes/unreleased.rst

```diff
@@ -69,6 +69,8 @@ None
 Changes
 =======
 
+- The Javascript user defined function language is now enabled by default.
```

Comment:
Should we maybe highlight that this does not affect the CE edition? (as JS UDF are only working on non-CE at all)